tests/end2end/test_activation_profile.py

```diff
@@ -46,6 +53,118 @@
 ACTPROF_VOLATILE_PROPS = []
 
 
+def standard_activation_profile_props(cpc, profile_name, profile_type):
+    """
+    Return the input properties for creating standard activation profile in the
+    specified CPC.
+    """
+    actprof_input_props = {
+        'profile-name': profile_name,
+        'description': (
+            '{} profile for zhmcclient end2end tests'.format(profile_type)),
+    }
+    if profile_type == 'image':
+        if cpc.prop('processor-count-ifl'):
+            actprof_input_props['number-shared-ifl-processors'] = 1
+        elif cpc.prop('processor-count-general-purpose'):
+            actprof_input_props['number-shared-general-purpose-processors'] = 1
+        else:
+            actprof_input_props['number-shared-general-purpose-processors'] = 1
+            warnings.warn(
+                "CPC {c} shows neither IFL nor CP processors, specifying 1 CP "
+                "for image activation profile creation.".
+                format(c=cpc.name), End2endTestWarning)
+
+        # actprof_input_props['processor-usage'] = 'shared'
```

Comment:
Why is this commented out?

Reply:
I had to experiment a bit until the creation worked. See the discussion in hmcse-dev. The active code is the minimum set of properties that succeeds, and left the others commented out.

I think I should remove the comments with the additional properties.